core/domain/acl_decorators.py

```diff
@@ -2386,6 +2386,18 @@ def generate_decorator_for_handler(handler):
         """
         def test_can_accept_suggestion(
                 self, target_id, suggestion_id, **kwargs):
+            """Returns a decorator handler to test whether a suggestion can be
+            accepted based on the user actions and roles.
+
+            Args:
+                target_id: str. The target id.
+                suggestion_id: str. The suggestion id.
+                **kwargs: *. Keyword arguments.
+
+            Returns:
+                function. The newly decorated handler to test the acceptance of
```

Comment:
No, there are two earlier return statements. I think you want something like: "The (possibly-decorated) handler for accepting a suggestion."

Reply:
Done